packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1854,6 +1875,7 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
     }
     for (NavigatorObserver observer in widget.observers)
       observer.didReplace(newRoute: newRoute, oldRoute: oldRoute);
+    _sendRouteChangeToSystemChannels('pushRoute', newRoute, oldRoute);
```

Comment:
can pushRoute be defined as a const somewhere?

Reply:
Done.